lib/my_john_deere_api/helpers/case_conversion.rb

```diff
@@ -22,16 +22,19 @@ def underscore(something)
   def camelize(something)
     something = something.to_s if something.is_a?(Symbol)
 
-    if something.is_a?(String)
+    case something
+    when String
       list = something.strip.split(/[_\s]+/)
 
       # preserve case of the first element
       new_list = [list.shift]
       new_list += list.map(&:capitalize)
 
       new_list.join('')
-    elsif something.is_a?(Hash)
+    when Hash
       something.transform_keys{ |key| camelize(key) }
+    when Array
+      something.map{|element| camelize(element)}
```

Comment:
I think you should put `Symbol` into this case statement and use the same `String` logic but add `something.to_s` to it.

Reply:
Haha, I missed that one, thanks :)